autoload/db.vim

```diff
@@ -206,7 +206,7 @@ if !exists('s:results')
 endif
 
 function! s:init() abort
-  setlocal nowrap nolist readonly nomodifiable nobuflisted bufhidden=delete
+  setlocal nowrap nolist readonly nomodifiable nobuflisted bufhidden=delete filetype=sql
```

Comment:
The contents of the buffer aren't SQL, so a file type of SQL doesn't make sense.

Reply:
You're right. However, it works quite well in practice for output coming from postgres/Redshift.

Reply:
You can do `autocmd BufReadPost *.dbout setf sql` if you like.